cli/cmd/root.go

```diff
@@ -43,6 +44,8 @@ func Execute() {
 	currentLogLevel := logging.LogLevel
 	logging.LogLevel = logging.QuietLevel
 
+	runVersionCheck()
```

Comment:
any chance to NOT run the version check if `--help` or similar is used?

Reply:
I added the check, if `help` occurs in command arguments, the version check will not be executed.